server/jetty/src/main/java/io/deephaven/server/jetty/JettyServerModule.java

```diff
@@ -67,9 +67,15 @@ static ServletAdapter provideGrpcServletAdapter(
         return serverBuilder.buildServletAdapter();
     }
 
+    @Binds
+    JsPluginRegistration bindJsPlugins(JsPlugins plugins);
+
     @Provides
-    @Named(PluginsModule.JS_PLUGIN_CONSUMER_NAME)
-    static Consumer<JsPlugin> providesJsPluginConsumer(JettyBackedGrpcServer server) {
-        return server.jsPluginConsumer();
+    static JsPlugins providesJsPluginRegistration() {
```

Comment:
These also add `JsPlugins` to the public API.

Reply:
Ok, seems like making it public is the "better" option?